packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -210,7 +222,7 @@ function Title( { item } ) {
 						// See https://github.com/WordPress/gutenberg/pull/51898#discussion_r1243399243.
 						tabIndex="-1"
 					>
-						{ item.title || item.name }
+						{ title || item.name }
```

Comment:
Should we `decodeEntities` here to include `item.name`?

Reply:
As far as I've tested, the `name` gets those entities removed, so we don't need to do that.

For example, creating a pattern with title `andr&eacute;` creates the following:

- id: 1838
- name: `andr`
- title: `andr&eacute;`